app/components/post_body/post_body.js

```diff
@@ -414,15 +412,11 @@ export default class PostBody extends PureComponent {
         if (!hasBeenDeleted) {
             body = (
                 <View style={style.messageBody}>
-                    <ScrollView
+                    <View
```

Comment:
I made a mistake here and did not put this back. Going to submit another PR as soon as I can to revert this change here. FYI @amyblais